backend/pick-git/src/main/java/com/woowacourse/pickgit/post/presentation/PostController.java

```diff
@@ -0,0 +1,34 @@
+package com.woowacourse.pickgit.post.presentation;
+
+import com.woowacourse.pickgit.authentication.domain.Authenticated;
+import com.woowacourse.pickgit.authentication.domain.user.AppUser;
+import com.woowacourse.pickgit.post.application.CommentRequestDto;
+import com.woowacourse.pickgit.post.application.CommentResponseDto;
+import com.woowacourse.pickgit.post.application.PostService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api")
+public class PostController {
+
+    private final PostService postService;
+
+    public PostController(PostService postService) {
+        this.postService = postService;
+    }
+
+    @PostMapping("/posts/{postId}/comments")
+    public ResponseEntity<CommentResponseDto> addComment(@Authenticated AppUser appUser,
+        @PathVariable Long postId, @RequestBody String content)
```

Comment:
```suggestion
    public ResponseEntity<CommentResponseDto> addComment(
        @Authenticated AppUser appUser,
        @PathVariable Long postId,
        @RequestBody String content)
```

이렇게 개행하면 가독성이 더 좋을 거 같습니다 !